### DISCUSSION TITLE
Contributing to the rust migration

### DISCUSSION THREAD

**Discussion by sladyn98** [vercel/turborepo#4823] [Help]
Hey I am a new contributor and would love to contribute to the rust migration. I checked for issues on github but I did not find much, Is there a place where I can start contributing. It could even be with small issues

**Comment by mehulkar** [comment]
We may not specifically have Rust migration work available, but check out the [`good first issue`](https://github.com/vercel/turbo/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label for where you can help! (cc @anthonyshew)
